lib/lita/handlers/pagerduty_utility.rb

```diff
@@ -96,7 +96,7 @@ def forget(response)
       private
 
       def lookup_on_call_user(schedule_id)
-        now = Time.now.utc
+        now = Time::Time.now.utc
```

Comment:
Thanks for the feedback -- you are right, of course. I've just changed it to this.